google/cloud/bigtable/admin_client.cc

```diff
@@ -144,13 +147,17 @@ class DefaultAdminClient : public bigtable::AdminClient {
 };
 }  // anonymous namespace
 
+namespace google {
+namespace cloud {
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
 std::shared_ptr<AdminClient> CreateDefaultAdminClient(
-    std::string project, bigtable::ClientOptions options) {
+    std::string project, google::cloud::bigtable::ClientOptions options) {
```

Comment:
Since we're already in the `google::cloud::bigtable` namespace here, should this be explicit about using `::google::cloud::bigtable` namespace to refer to the class, or is that unnecessary?

Reply:
Actually being explicit confuses Doxygen, it thinks that this is an overload because the function declaration uses a different spelling of `ClientOptions`.  Thanks for getting me to look into this.